package.json

```diff
@@ -9,7 +9,7 @@
   "author": "Jim Cramer",
   "license": "CC-BY-4.0",
   "devDependencies": {
-    "eslint": "^5.9.0",
+    "eslint": "^5.16.0",
```

Comment:
Why was this updated?

Reply:
when i run npm install it got automatically modified.